README.md

```diff
@@ -16,3 +16,23 @@ copyright and related rights in the work worldwide are waived through
 the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
 
 For more information, see [license](LICENSE.md).
+
+
+## Deployment
+
+Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting set up with Cloud Foundry
+
+
+To deploy:
+`cf target -o 18f -s dolores` 
+
+cf set-env dolores HOST dolores.18f.gov
```

Comment:
This won't work with the proxying we're setting up in 18F/hub#441, as that's the URL we want team members to use, hence the CF hostname needs to be different. `dolores-app.18f.gov` would be a possibility.